analytics/analytics.go

```diff
@@ -25,6 +25,9 @@ type Analytics struct {
 
 // Track submits an event to the identity service
 func (a *Analytics) Track(ctx context.Context, name string, params *map[string]string) error {
+	if !a.session.IsUser() {
+		return nil
```

Comment:
All of our analytics depend on tracking actions on a team as a user. Segment won't let us track anything just based on a team alone.

@maybecolin can suggest otherwise, but having this subset of automated activity missing is ok imo.

Reply:
If anything server-side tracking could be added. Nothing in CLI.

Reply:
https://github.com/manifoldco/engineering/issues/3211

Reply:
:+1: